install/webui-user.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Is this file used anywhere? It's probably meant to be an override to the default webui-user.sh contained in AUTOMATIC1111 project - if so, please move it under docker/prep and ensure it's overwriting it in Dockerfile.

Correct me though if my understanding there is wrong.

Reply:
Yeah, this file is used on provision step (install.ps1:93) to add custom command line arguments and . I left it in `install` folder as it's not included in Docker image and copied directly to the container (so user can change command line parameters without rebuilding Docker image). If you want, I can move it to `docker/prep`